README.md

```diff
@@ -47,6 +48,20 @@ Execute standalone JAR with `java`:
 - `-o`/`--outdir`: Output directory; where to create the `<workflow-name>/<workflow-version>/` directory structure and write the `irida_workflow.xml`, `irida_workflow_structure.ga` and `messages_en.properties` files
 - `-x`/`--extra-tool-param-attrs`: Save extra toolParameter attributes `["label", "type"]` to XML `toolParameter` tags
 - `--remove-output-name-file-ext`: Remove file extension in workflow output names? (default is to keep the extensions in the `<output>` tag names)
+- `-u`/`--uuid-as-toolid`: Use UUID when building tool_ids, needed for workflows with duplicate tools.
```

Comment:
I'm wondering if we should maybe invert this? That is using uuid is the default and the option should be `--use-tool-name-as-id` or something like that?

Reply:
I actually prefer the parameter name to stay as is. It ends up producing something that looks like:

```xml
    <parameter name="refseq_masher-1-adv.top_n_results" defaultValue="5">
      <toolParameter toolId="163f0c35-99ff-4b90-8f16-0a76eb44315b" parameterName="adv.top_n_results"/>
    </parameter>
...
    <parameter name="refseq_masher-2-adv.top_n_results" defaultValue="0">
      <toolParameter toolId="9f708acc-1c4f-4420-8d2c-49e2462a9340" parameterName="adv.top_n_results"/>
    </parameter>
```

That is, it makes it much easier to see what the particular parameter corresponds too in the XML file. Changing everything to the UUID would mean that someone would have to have open both the XML and the Galaxy `.ga` file and match entries up to figure out what parameter matched with what tool.

Note that the name in `<parameter name=..>` is only used to match up with the corresponding entry in the `messages_en.properties` file. It's not displayed anywhere.

My reason for wanting it inverted is that I believe in general, tools should just work with the default parameters for the most number of cases. Leaving `--uuid-as-toolid` as a parameter means that if anyone has duplicate tools in their workflow they will first have to run `irida-wf-ga2xml`, have it crash, dig through documentation (or post an issue) as to why, before figuring out they need to add `--uuid-as-toolid`. If using UUID's is enabled by default, then `irida-wf-ga2xml` will just work in more cases.